### DISCUSSION TITLE
Disabling Integrations

### DISCUSSION THREAD

**Discussion by m4444x** [frenck/spook#519] [Q&A] [answered]
Hi @frenck !

I got a few questions for you about the `homeassistant.disable_config_entry` service :

1. Will it work on all integrations ( including those in the default config category ) ?
2. Will it allow me to disable the Spook integration itself ?
3. Are the integrations disabled forever or is the effect temporary ( like until the next reboot ) ?

**Comment by frenck** [comment, marked as answer]
1. Any config entry (so entity integration that is set up from the UI).
2. Yes.
3. Until you enable it again (either via the enable Spook service, or manually in the UI)
